modules/configuring-ovnk-additional-networks.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_content-type: CONCEPT
+[id="configuration-ovnk-additional-networks_{context}"]
+= Configuration for an OVN-Kubernetes additional network
+
+The {openshift-networking} OVN-Kubernetes network plugin can be configured to allow pods to have secondary network interfaces. To configure secondary network interfaces, you must define the configurations in the `NetworkAttachmentDefinition` custom resource definition (CRD).
+
+:FeatureName: Configuration for an OVN-Kubernetes additional network
+include::snippets/technology-preview.adoc[]
+
+The following sections provide example configurations for each of the topologies that OVN-Kubernetes currently allows for secondary networks.
+
+[NOTE]
+====
+Networks names must be unique. For example, creating multiple `NetworkAttachmentDefinition` CRDs with different configurations that point at the same network is unsupported.
+====
+
+[id="configuration-ovnk-network-plugin-json-object"]
+== OVN-Kubernetes network plugin JSON configuration object
+
+The following object describes the configuration parameters for the OVN-Kubernetes CNI network plugin:
+
+[cols=".^2,.^2,.^6",options="header"]
+|====
+|Field|Type|Description
+
+|`cniVersion`
+|`string`
+|The CNI specification version. The `0.3.1` value is required.
+
+|`name`
+|`string`
+|The name of the network. This value must be unique across all `NetworkAttachmentDefinitions`. Failure to make this attribute unique might result in the merging of two networks with the same name. Merging two networks by sharing similar names is unsupported.
+
+|`type`
+|`string`
+|The name of the CNI plugin to configure: `ovn-k8s-cni-overlay`.
+
+|`topology`
+|`string`
+|The topological configuration for the network. The value must be `layer2`.
+
+|`subnets`
+|`string`
+| The subnet to use for the network across the cluster. If you specify `layer2` for the `topology`, only include the CIDR for the node. For example, `10.100.200.0/24`.
+
+For `"topology":"layer2"` deployments, IPv6 and dual-stack subnets are supported. For example, `192.168.100.0/24,fd00:dead:beef::0/64`.
```

Comment:
If you want to use the official [IPv6 Address Prefix Reserved for Documentation](https://www.rfc-editor.org/rfc/rfc3849), it's the IPv6 address prefix `2001:DB8::/32`

Similarly, [IPv4](https://www.rfc-editor.org/rfc/rfc5737.html) has the blocks `192.0.2.0/24` (TEST-NET-1), `198.51.100.0/24` (TEST-NET-2), and `203.0.113.0/24` (TEST-NET-3).

I fully recognize that I am a giant nerd for thinking about this :nerd_face: